hy/cmdline.py

```diff
@@ -293,8 +288,27 @@ def proc_opt(opt, arg=None, item=None, i=None):
             except HyLanguageError:
                 hy_exc_handler(*sys.exc_info())
                 sys.exit(1)
+    elif action == "just_repl":
+        pass
+    else:
+        assert False
 
-    return REPL(spy=options.get("spy"), output_fn=options.get("repl_output_fn")).run()
+    assert repl
```

Comment:
redundant? In any case, raising an AssertionError here seems no better to me than raising an AttributeError just a few more lines down, since we're not trying to catch either of them higher up anywhere.

On the other hand, if it's for semantically documenting that we're definitely setting up the REPL with this combination of options, then I would replace this with an explicit comment stating as such.

Reply:
Sure, I replaced it with a comment.